Include/cpython/abstract.h

```diff
@@ -4,10 +4,6 @@
 
 /* === Object Protocol ================================================== */
 
-#ifdef PY_SSIZE_T_CLEAN
-#  define _PyObject_CallMethodId _PyObject_CallMethodId_SizeT
-#endif
-
```

Comment:
`_PyObject_CallMethodId_SizeT` is removed and `_PyObject_CallMethodId` is ssize_t version.